src/klein/storage/_memory.py

```diff
@@ -55,7 +55,7 @@ class _MemoryAuthorizerFunction:
     Type shadow for function with the given attribute.
     """
 
-    __memoryAuthInterface__: IInterface = None
+    __memoryAuthInterface__: IInterface = None  # type: ignore[assignment]
```

Comment:
Don't allow this to be set to `None` outside of init.

Reply:
I generally agree, though I think None is fine in this context and don't see there need to define an `IUndeclared`.

I added an assertion above the line were `__memoryAuthInterface__` is accessed to make sure it isn't still `None`.